### ISSUE TITLE
there should be a shortcut for jumping between new and old files

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#188284]
Use "Diff Editor: Switch Side" command to jump from original to modified and vice versa. Notice how selections are mapped.

There is no keybinding for it, but users can configure their own.

**Comment by kupuma-ru21** [comment]
I made extension which should be useful for this.
https://marketplace.visualstudio.com/items?itemName=kupuma-ru21.Cursorghost
